blueocean-core-js/src/js/config.js

```diff
@@ -35,6 +36,10 @@ export default {
         return config.jenkinsConfig || {};
     },
 
+    getOrganization() {
+        return encodeURIComponent(organization.name);
```

Comment:
@cliffmeyers There are different ways to do it. We can either keep different method names or pass encoding true or false via method param. e.g.  getOrganization(encoded=true)?